clusters/lib/gatekeeper/application.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: gatekeeper
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/gatekeeper.git
+    targetRevision: HEAD
+    path: REPLACEME
+  destination:
+    name: REPLACEME
```

```suggestion
    name: nerc-ocp-prod
```

Not sure if this is the correct value here, just assuming from the gatekeeper-policy application manifest.